### PR TITLE
HOC: Add sign up emails for 2020

### PR DESCRIPTION
**Pull request by molly-moen**
Add new 2020 sign-up receipt emails for hour of code. The English one is updated for this year and the Spanish and Portuguese ones are copied from last year with new dates. Adding these will not change any behavior until we switch to `pre-hoc` mode and update the `hoc_year` to 2020. We will likely completely change the Spanish and Portuguese emails later on, once the new text is translated.

## Links

- [email text doc](https://docs.google.com/document/d/1Gcul-wfz8DUaiuyb6vpR48WUMk16RfbbE_biMMuujjg/edit#heading=h.6h602wg50amw)
- [jira](https://codedotorg.atlassian.net/browse/PLC-934)

## Testing story
Validated that when hoc_year is changed to 2020 the new emails get sent, and the html looks correct.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: pegasus/emails/hoc_signup_2020_receipt_en.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+Hour of Code kits, fun stickers, and more! But hurry, supplies are limited.
+
+### Encourage kids to continue learning 
+<% if form.data["hoc_event_country_s"] == 'US' %> An Hour of Code is just the beginning! We hear over and over again 
+how much students love the Hour of Code and discover a newfound interest in computer science. Encourage them to 
+continue learning. Whether you’re an administrator, teacher, or advocate, we have professional learning, curriculum, 
+and resources to help you [bring computer science classes to your school](https://<%= codedotorg %>/yourschool) or 
+expand your offerings.  
+
+Many of the organizations offering activities on HourofCode.com also have curriculum and professional learning
```
Comment (breville) [inline pegasus/emails/hoc_signup_2020_receipt_en.md:48]: I've not seen this capitalisation of hourofcode.com.  Is it our standard?

**Review by breville — APPROVED** [review]

File: pegasus/emails/hoc_signup_2020_receipt_en.md (hunk 1 above)
Comment (molly-moen) [reply pegasus/emails/hoc_signup_2020_receipt_en.md:48]: not sure, it was in the copy Mariah made

**Review by bencodeorg — APPROVED** [review]

**Comment by bencodeorg** [comment]
I wonder what the difference is between .md and .md.erb files, since this has ERB in it but is a markdown file?

**Comment by breville** [comment]
@bencodeorg, @Hamms put up some really good documentation at https://code.org/editing/concepts.

**Comment by bencodeorg** [comment]
Whoa this is awesome, thanks!

**Pull request merged**